civihr_default_theme/scss/base/_task-assignments-modal-temp.scss

```diff
@@ -8,15 +8,20 @@ $lighter-grey: #ECEDEE;
 $white: #FFFFFF;
 
 // Opened Modal
-.modal-open {
+[data-ta-documents],
+[data-ta-documents-manager] {
   .modal {
     overflow-y: scroll !important;
 
     .modal-dialog {
-      width: 500px;
+      max-width: 500px;
```

Comment:
Actually, the modal is being reused form T&A and works fine  in T&A. But the styles are not present in SSP as we are using different theme than T&A. So the width needs to be displayed s above, which is temporary for now.